<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1872</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Admin">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Admin</string>
    </property>
    <widget class="QLineEdit" name="testBattery">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>40</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery Level</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="percentage">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>171</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Test Buttons</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="maxBattery">
         <property name="text">
          <string>MAXIMUM POWA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lowBattery">
         <property name="text">
          <string>Low Battery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="noBattery">
         <property name="text">
          <string>No Battery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectPc">
         <property name="text">
          <string>Connect with PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateSignal">
         <property name="text">
          <string>Generate Signal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eegToggle">
         <property name="text">
          <string>EEG Toggle</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin - Device Test</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>451</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeviceView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>811</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Device View</string>
    </property>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>401</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>251</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="newSession">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>New Session</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menu">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>481</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>New Session View</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Session Info</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="sessionInfo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="contact">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sessionLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Session Log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitTime">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>220</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="power">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>on/off</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="battery">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>410</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ComputerView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>811</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Computer View</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Computer View</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="logList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>511</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1872</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNeuroNet_UI">
    <property name="title">
     <string>NeuroNet UI</string>
    </property>
   </widget>
   <addaction name="menuNeuroNet_UI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
